mmpose/apis/inference_tracking.py

```diff
@@ -36,7 +36,7 @@ def _compute_iou(bboxA, bboxB):
     return iou
 
 
-def _track_by_iou(res, results_last, thr):
+def _track_by_iou(res, results_last, thr, **kwargs):
```

Comment:
I would recommend not using ``**kwargs`` but it seems there is no other simple solution here. @jin-s13 Do you have any suggestions?

Reply:
Maybe we can use `functools.partial` to unify the interfaces

Reply:
> Maybe we can use `functools.partial` to unify the interfaces

Could you please give an example?

Reply:
> Could you please give an example?

```python
def _track_by_iou(res, results_last, thr): 
    # track by iou

def _track_by_oks(res, results_last, thr, sigmas): 
    # track by oks

_track = partial(_track_by_oks, sigmas=dataset_sigmas) if use_oks else _track_by_iou
```

if `use_oks` is True, then `_track` acts like the following function
```python
def func(res, results_last, thr): 
    return _track_by_oks(res, results_last, thr, dataset_sigmas)
```

Reply:
Good idea. I've already modified with this method :).